NOTICE

```diff
@@ -0,0 +1,156 @@
+This content is produced and maintained by the Eclipse SmartHome project.
+
+* Project home: https://eclipse.org/smarthome/
+
+== Declared Project Licenses
+
+This program and the accompanying materials are made available under the terms
+of the Eclipse Public License 2.0 which is available at
+http://www.eclipse.org/legal/epl-2.0.html.
```

Comment:
This is giving me a 404 error, correct link is: https://www.eclipse.org/legal/epl-2.0/